### DISCUSSION TITLE
Running Dependabot as a GitHub Action workflow and Renovate?

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#28922] [Request Help] [answered]
### Tell us more.

Quote from the GitHub Blog:

> Starting today, administrators using `Github.com` accounts can enable their repositories and/or organizations to run Dependabot updates jobs as a GitHub Actions workflow using both hosted and self-hosted runners. **Running Dependabot does not count towards GitHub Actions minutes–meaning that using Dependabot continues to be free for everyone.**

Do we (as maintainers) need to do anything to support Dependabot vulnerability alerts? I think we need Dependabot vulnerability alerts for our [`vulnerabilityAlerts` config option](https://docs.renovatebot.com/configuration-options/#vulnerabilityalerts) at least. I guess more Renovate things rely on getting the correct info from Dependabot?

Sources:

- [GitHub Blog, Dependabot on GitHub Actions and self-hosted runners is now generally available](https://github.blog/2024-05-02-dependabot-on-github-actions-and-self-hosted-runners-is-now-generally-available/)
- [GitHub Docs, About Dependabot on GitHub Actions runners](https://docs.github.com/en/code-security/dependabot/working-with-dependabot/about-dependabot-on-github-actions-runners)

**Comment by rarkins** [comment, marked as answer]
Dependabot _alerts_ are calculated by GitHub independently of Dependabot itself, in a backend unrelated to GitHub Actions, so not related to this change.
